Protocol Buffer

traxys opened the pull request:
Some functions used to communicate beetween processes.

traxys committed:
Initial protocols

ghost commented:
Il semblerait que le seul module implémenté soit le mineur. Malheureusement, nous ne pouvons accepter dans le standard un travail aussi incomplet.
Avant de valider cette pull request, il va donc falloir que nous ajoutions plus de messages. Il faut par exemple pouvoir supporter un wallet.

traxys committed:
Cleanup

traxys committed:
Version tag

traxys committed:
Idk how to proto

traxys committed:
Packages

traxys committed:
Module and name

Commit:
Ajout de trois messages et suppression de Reply.

Ajout de :

- GetInfo
- GetBlockByHash
- GetTxByHash

Reply n’est désormais plus utilisé.

ghost commented:
Après réflexion, je trouve que la structure `Reply` rend plus difficile une mise à jour future. Je l’ai donc enlevée dans mon commit, qu’en pensez-vous ?

traxys committed:
Adaptator

traxys committed:
Merge branch 'gRPC' of github.com:EnsicoinDevs/ensicoin into gRPC

traxys committed:
meager fix

traxys committed:
Why stream
